pgptool-gui/src/main/java/org/pgptool/gui/tools/fileswatcher/MultipleFilesWatcher.java

```diff
@@ -37,230 +38,84 @@
 import org.apache.commons.io.FilenameUtils;
 import org.apache.log4j.Logger;
 
-import ru.skarpushin.swingpm.modelprops.virtualtable.RecurringBackgroundTask;
-
-// NOTE: Consider migrating to org.apache.commons.io.monitor.FileAlterationObserver, it appears we introduced bycle here
```

Comment:
All these changes were made as your dependency resolution is broken and you had the TODO anyway so even fixing above error is not enough.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.1:compile (default-compile) on project gui: Compilation failure: Compilation failure: 
[ERROR] /home/fred/Workspace/pgptool/pgptool-gui/src/main/java/org/pgptool/gui/tools/fileswatcher/MultipleFilesWatcher.java:[42,52] error: cannot find symbol
[ERROR] 
[ERROR]  package ru.skarpushin.swingpm.modelprops.virtualtable
[ERROR] /home/fred/Workspace/pgptool/pgptool-gui/src/main/java/org/pgptool/gui/tools/fileswatcher/MultipleFilesWatcher.java:[59,9] error: cannot find symbol
[ERROR] 
[ERROR]  class MultipleFilesWatcher
[ERROR] /home/fred/Workspace/pgptool/pgptool-gui/src/main/java/org/pgptool/gui/tools/fileswatcher/MultipleFilesWatcher.java:[149,5] error: cannot find symbol
[ERROR] 
[ERROR]  class BaseFolder
[ERROR] /home/fred/Workspace/pgptool/pgptool-gui/src/main/java/org/pgptool/gui/tools/fileswatcher/MultipleFilesWatcher.java:[151,38] error: cannot find symbol
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```